src/controller/datacontroller.js

```diff
@@ -306,6 +335,57 @@ export default class DataController {
 
 mix( DataController, EmitterMixin );
 
+// Traverses given DocumentFragment and searches elements which marks marker range. Founded element is removed from
+// DocumentFragment but path of this element is stored in Map.
+//
+// @param {module:engine/view/documentfragment~DocumentFragment} documentFragment Model DocumentFragment.
+// @returns {Object} Object with markers data and cleaned up document fragment.
+function extractMarkersFromModelFragment( documentFragment ) {
```

Comment:
Finally, this could be a private method too, but I also have this "problem" of using sometimes private "static-like" methods and sometimes not exported functions, so as you wish :)